netbox/netbox/settings.py

```diff
@@ -102,6 +102,7 @@
 PLUGINS_CONFIG = getattr(configuration, 'PLUGINS_CONFIG', {})
 RELEASE_CHECK_URL = getattr(configuration, 'RELEASE_CHECK_URL', None)
 REMOTE_AUTH_AUTO_CREATE_USER = getattr(configuration, 'REMOTE_AUTH_AUTO_CREATE_USER', False)
+REMOTE_AUTH_AUTO_CREATE_GROUP = getattr(configuration, 'REMOTE_AUTH_AUTO_CREATE_GROUP', False)
```

Comment:
```suggestion
REMOTE_AUTH_AUTO_CREATE_GROUPS = getattr(configuration, 'REMOTE_AUTH_AUTO_CREATE_GROUPS', False)
```

I think you made a copy paste error. When changing it to the above suggestion the PR works as intended.

Reply:
You should also add the new setting to the docs.